dev-docs/source/Packaging.rst

```diff
@@ -0,0 +1,124 @@
+
+Each package is built separately for ``x64`` Windows, Linux and macOS with the
+exception of ``mantiddocs`` where a single, ``noarch`` package is created for all
+operating systems as it only contains static HTML.
+
+All mantid-conda packages are available to users through the Mantid's
+`Conda organization <mantid-conda-org>`_.
+
+Building Conda Packages
+#######################
+
+Conda requires `recipes <conda-recipes-docs_>`_ as input for producing packages.
+The recipes are stored in the `main codebase <mantid-conda-recipes_>`_ so that
+they can evolve as the code develops. The packages are built as part of the
+`CI pipeline <ci-pipeline_>`_ using a `script <package-conda_>`_ to encapsulate
+the steps.
+
+To build the packages locally it is recommended that you use a separate
+clone of the repository as the build copies the repository content to a temporary
+working directory and any additional build directories can interfere with the
+packaging. The following steps should produce a set of conda packages in a ``conda-bld``
+directory relative to the current working directory
+(on Windows run this from Git bash):
+
+.. code:: sh
+
+   git clone https://github.com/mantidproject/mantid.git mantid-conda-build
+   mantid-conda-build/buildconfig/Jenkins/Conda/package-conda $PWD \
+     --build-mantid --build-qt --build-docs --build-workbench 2>&1 | tee package.log
+   # wait quite a while ...
+   # packages will appear in a conda-bld directory
+
+To create a new test enviroment with packages from the local build:
+
+.. code:: sh
+
+   mamba create -n local-package-test -c $PWD/conda-bld mantidworkbench
+   mamba activate local-package-test
+   workbench
+
+
+Standalone
+----------
+
+Mantid also provides a set of bundled packages that provide a complete install of
+``MantidWorkbench`` and all of its dependencies without a user having to
+first setup conda and then install the relevant conda packages.
+The standalone packages are available to users through Mantid's `download page <download-page_>`_.
+
+Each installer is simply a conda environment, with ``mantidworkbench`` installed within
+it followed by the appropriate installer technology wrapped around it.
+Installers are provided for:
+
+- `Windows`: A `NSIS <nsis_>`_ installer package providing a wizard-style interface
+  for installing/uninstalling MantidWorkbench.
+- `macOS`: A `disk image <dmg_>`_ providing a facility to drag-and-drop MantidWorkbench
+  to the `/Applications` folder
+- `Linux`: An ``xz``-compressed tarball of the package files and directories that
+  can be extracted to any location.
+
+Building Standalone Installers
+##############################
+
+The installers are built as part of the `CI pipeline <ci-pipeline_>`_ using a
+`script <package-standalone_>`_ to encapsulate the build steps.
+
+To build an installer locally first build a set of conda packages using the
+`instructions above <building-conda-packages_>`_. Once you have the ``conda-bld``
```

Comment:
internal reference not working for me
```suggestion
:ref:`instructions above <building-conda-packages>`. Once you have the ``conda-bld``
```
which also needs this above the `Building Conda Packages` header:
```
.. _building-conda-packages:

```

That works for me, but there's probably another way to do it.